_posts/2018-08-27-sharding-bitcoin-cash.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: Sharding Bitcoin Cash
+subtitle: Article by Shammah Chancellor, Bitcoin ABC Developer
+multiLangId: 2018-08-27-sharding-bitcoin-cash
```

Comment:
Lets not backdate this. Please change 8-27 to 9-6 here and in the file name.

Reply:
OK, l I have made the change